discovery-provider/plugins/notifications/src/email/notifications/sendEmail.ts

```diff
@@ -102,6 +102,26 @@ export const sendNotificationEmail = async ({
   }
 }
 
+export const sendTransactionalEmail = async ({ email, html, subject }) => {
+  try {
+    logger.debug(`SendTransactionalEmail | ${email}, ${subject}`)
+    const emailParams = {
+      from: 'Audius <team@audius.co>',
+      to: `${email}`,
+      html,
+      subject,
+      asm: {
+        groupId: 23583 // id of unsubscribe group at https://mc.sendgrid.com/unsubscribe-groups
```

Comment:
nit, can we make this a top level const or env var? 
`const UNSUBSCRIBE_GROUP_ID = parseInt(process.env.UNSUBSCRIBE_GROUP_ID || "23583") ;

Reply:
added as a const, borrowed from the existing pattern here, so updated that too.